### PR TITLE
Enable dune wrapped in transition mode

### PR DESCRIPTION
**Pull request by wokalski**
Fixes https://github.com/janestreet/incr_dom/issues/5
Fixes https://github.com/janestreet/virtual_dom/issues/3

### CODE REVIEW

File: lib/js_of_ocaml.ml [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  *)
 
-module CSS = CSS
+module CSS = CSS__Internal
```
Comment (rgrinberg) [inline lib/js_of_ocaml.ml]: You should not use `__` in your own module names. This is just a naming convention that's taken up by dune's wrapping mechanism. You should use some name like `css_internal` or w/e

**Comment by rgrinberg** [comment]
Note that this is a breaking change. As it means that the CSS module will no longer accessible without the alias.

File: lib/js_of_ocaml.ml (hunk 1 above)
Comment (wokalski) [reply lib/js_of_ocaml.ml]: @hhugo is there any name for the module you’d prefer?

**Comment by hhugo** [comment]
I would prefer to do the change in two steps, and maybe do a release in between.
1. enable to wrapping in dune
2. rename the css module (cSS0.ml maybe)

**Comment by rauanmayemir** [comment]
@hhugo Step 1 is done. If you merge it, I'll add the rename on top of it.

**Comment by wokalski** [comment]
I think it might not have been clear. The change has been performed by @rauanmayemir. Please check and let me know if everything looks good.

**Comment by hhugo** [comment]
Thanks

**Pull request merged**